shell/probe/raindrop.ps1

```diff
@@ -1,5 +1,11 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
+function Log {
+    Param([String]$line)
+
+    Write-EventLog -LogName "Application" -Source "Prelude Probe Service" -EventId 0 -Message $line -ErrorAction Ignore
```

Comment:
Why a separate function vs just using this instead of write output?

Reply:
Using this Log function = 2003 chars
Copy pasting that same line = 2156 chars

Reply:
Good